lib/note-detail/index.tsx

```diff
@@ -238,7 +239,7 @@ const mapStateToProps: S.MapState<StateProps> = ({
   ui,
   settings,
 }) => ({
-  dialogs: state.dialogs,
+  dialogs: ui.dialogs,
```

Comment:
based on the usage inside the component I think it would be an appropriate update to change this from passing in the list of opened dialogs so that it passed in whether or not dialogs are open.

```ts
const mapStateToProps…
isViewingDialog: ui.dialogs.length > 0,
```

Reply:
I don't actually even know how to hit this code path